### ISSUE TITLE
Duplicate logs getting created 

### ISSUE THREAD

**Issue by LoydAntony** [DaveWoodCom/XCGLogger#327]
# Duplicate logs getting created 
## Issue 
When we added the log in the code, the logs are getting created twice. I have tried adding the log on Delegate its still printing the logs twice in the console.

- Xcode: 14.2 (Rosetta)
- XCGLogger version: 7.2.0
- swift: 5

<img width="869" alt="Screenshot 2023-06-09 at 8 11 59 PM" src="https://github.com/DaveWoodCom/XCGLogger/assets/98867104/da436fd4-d31f-4d83-aaef-f8eca1faf789">

`enabledDestinations.count` is 3 for one log registered which is printing two message on the console.

<img width="575" alt="Screenshot 2023-06-09 at 8 19 08 PM" src="https://github.com/DaveWoodCom/XCGLogger/assets/98867104/49fe6360-1f99-4d9b-aa8d-ed609e9d3423">

You can see the console the message `Theater loaded successfully 🎉 theater:` is printed twice on the console.
The LOG is triggered only once in the code.
 But in the XCLogger Class the function 
 ``` swift
 open func logln(_ level: Level = .debug, functionName: String = #function, fileName: String = #file, lineNumber: Int = #line, userInfo: [String: Any] = [:], closure: () -> Any?)
```
 is creating duplicate logs in the terminal, which is causing issue in debugging the source code.

It looks like there is bug in the login function which is causing the logs to get generated twice.

## Required Result
Print the log only once in the terminal and do not create duplicate of it

**Comment by sanojkashyap007** [comment]
Even I am also facing the same issue.
